src/app/feature/test/pages/test/test.component.html

```
@@ -0,0 +1,108 @@
+							</div>
+						}
+
+						<div class="navigation">
+							<button
+								class="nav-btn nav-btn--secondary"
+								[disabled]="currentQuestionIndex() === 0"
+								(click)="previousQuestion()"
+							>
+								Назад
+							</button>
+							@if (currentQuestionIndex() < test.data.questions.length - 1) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)